backend/clickhouse/logs.go

```diff
@@ -54,6 +54,9 @@ func (l *LogRow) Cursor() string {
 }
 
 func (client *Client) BatchWriteLogRows(ctx context.Context, logRows []*LogRow) error {
+	if len(logRows) == 0 {
```

Comment:
Is this really needed?

Reply:
shouldn't we avoid creating an empty batch if there are no rows? i'd rather support this implementation-specific logic here than wrap calls to `BatchWriteLogRows` with the check

Reply:
gotcha, in https://github.com/highlight/highlight/blob/93cc114c6d5e70f76abacb2200bad7a9358c67b5/backend/worker/kafka_worker.go#L79 our batch processing will batch different message types together adn invoke the clickhouse write if the logs slice is nil (could be no logs in a batch but batch still has `markbackendsetup` messages)